src/bsonjs.c

```diff
@@ -42,8 +43,20 @@ bson_str_to_json(const char *bson, size_t bson_len, size_t *json_len)
         bson_reader_destroy(reader);
         return NULL;
     }
-
-    json = bson_as_json(b, json_len);
+    if(mode == 1){
+        json = bson_as_relaxed_extended_json(b, json_len);
+    }
+    else if(mode == 2){
+        json = bson_as_canonical_extended_json(b, json_len);
+    }
+    else if(mode == 0){
+        json = bson_as_json(b, json_len);
+    }else{
```

Comment:
Formatting of this if/else if/else block is still inconsistent.

Reply:
This is still not resolved. Following the existing if/else statements in this file it should be:
```
    if (mode == 1) {
        json = bson_as_relaxed_extended_json(b, json_len);
    } else if(mode == 2) {
        json = bson_as_canonical_extended_json(b, json_len);
    } else if(mode == 0) {
        json = bson_as_json(b, json_len);
    } else {
        ...
    }
```